src/runtime/can_use_target.cpp

```diff
@@ -12,23 +14,27 @@ WEAK halide_can_use_target_features_t halide_set_custom_can_use_target_features(
     return result;
 }
 
-WEAK int halide_can_use_target_features(uint64_t features) {
-    return (*custom_can_use_target_features)(features);
+WEAK int halide_can_use_target_features(int count, const uint64_t *features) {
+    return (*custom_can_use_target_features)(count, features);
 }
 
-WEAK int halide_default_can_use_target_features(uint64_t features) {
+WEAK int halide_default_can_use_target_features(int count, const uint64_t *features) {
     // cpu features should never change, so call once and cache.
-    static bool initialized = false;
-    static CpuFeatures cpu_features;
-    if (!initialized) {
-        cpu_features = halide_get_cpu_features();
-        initialized = true;
+    static CpuFeatures cpu_features = halide_get_cpu_features();
```

Comment:
Looks like this is a problem, because CpuFeatures has a constructor now this static requires locking/unlocking to initialize, which isn't present in all environments. (cxa_guard errors in the windows build)

Reply:
...good thing we waited on the buildbots :-)